templates/index.html.hbs

```diff
@@ -153,7 +153,7 @@
         <!--==========================
           Team Section
         ============================-->
-        <section id="team">
+        {{!-- <section id="team">
```

Comment:
Please remove instead of commenting. We still have git's history if we need this code again. 

Commenting code, except in very specific cases, isn't a good practice in my opinion.